text/1000-migrations.rst

```diff
@@ -332,47 +359,88 @@ The returned JSON conforms to the following pseudo-schema::
 Example::
 
     {
+      "parent": "m1vrzjotjgjxhdratq7jz5vdxmhvg2yun2xobiddag4aqr3y4gavgq",
+
+      "complete": false,
+
       "confirmed": [
         "CREATE TYPE User { CREATE PROPERTY name -> str }"
       ],
 
-      "proposed": [
-        {
-          "statements": [{
-            "text": "ALTER TYPE Address " +
-                    "ALTER PROPERTY number " +
-                    "SET TYPE int64 USING \(expr)",
-            "required-user-input": [{
-              "name": "expr",
-              "prompt": "Altering Address.number to type " +
-                        "int64 requires an explicit conversion expression",
-            }]
-          }],
-          "confidence": 0.6
-          "prompt": "Did you alter the Address.number property?"
-        }, {
-          "statements": [{
-            "text": "ALTER TYPE Address { " +
-                    "DROP PROPERTY number; " +
-                    "CREATE PROPERTY number -> int64; }"
-          }],
-          "confidence": 0.4
-        }
-      ]
+      "proposed": {
+        "statements": [{
+          "text": "ALTER TYPE Address " +
+                  "ALTER PROPERTY number " +
+                  "SET TYPE int64 USING \(expr)",
```

Comment:
By the way, we don't have the description of this variable substitution, do we? Does CLI validate them somehow? (at least lexemes and parenthesis?) Are there constant substitutions where we can validate that variable is of the valid type, or is it just expression? Can there be any other parts of statements except expressions?

Reply:
The CLI should not attempt to interpret expressions or substitutions, treat it as string formatting.  Validation is done on the server.

Reply:
Added a clarification.